dask/array/core.py

```diff
@@ -3010,37 +3014,80 @@ def normalize_chunks(chunks, shape=None, limit=None, dtype=None, previous_chunks
 
     >>> normalize_chunks((), shape=(0, 0))
     ((0,), (0,))
+
+    Use ``numblocks`` to replicate a chunk shape a given number of times.
+    Note that ``'auto'``, -1, and None cannot be used if only ``numblocks`` is given.
+
+    >>> normalize_chunks(4, numblocks=(2, 3))
+    ((4, 4), (4, 4, 4))
+
+    Use ``shape`` and ``numblocks`` without any chunksize
+
+    >>> normalize_chunks(None, shape=(10, 5), numblocks=(2, 2))
+    ((5, 5), (3, 2))
     """
```

Comment:
It might be helpful to move some error handling to the very top. Like if all of `numblocks`, `shape`, and `chunks` are None, that is invalid. Or if `numblocks` and `shape` are None and chunks is not an iterable. Basically I think if you can separate out the error handling from the transformation that would be easier to parse.